Detectors/TPC/qc/src/Tracks.cxx

```diff
@@ -130,23 +138,46 @@ bool Tracks::processTrack(const o2::tpc::TrackTPC& track)
     auto propagator = o2::base::Propagator::Instance(true);
     const int type = (track.getQ2Pt() < 0) + 2 * track.hasCSideClustersOnly();
     auto dcaHist = mMapHist[fmt::format("hDCAr_{}", types[type]).data()].get();
+    const std::string signType((sign < 0) ? "Neg" : "Pos");
+    auto dcaHist_pT = mMapHist["hDCArVsPt" + signType].get();
+    auto dcaHist_eta = mMapHist["hDCArVsEtaPos" + signType].get();
+    auto dcaHist_nCluster = mMapHist["hDCArVsNClsPos" + signType].get();
```

Comment:
Should be camel case, see https://rawgit.com/AliceO2Group/CodingGuidelines/master/naming_formatting.html#Variable_Names

```suggestion
    auto dcaHistPT = mMapHist["hDCArVsPt" + signType].get();
    auto dcaHistEta = mMapHist["hDCArVsEtaPos" + signType].get();
    auto dcaHistNCluster = mMapHist["hDCArVsNClsPos" + signType].get();
```

Reply:
Changed it here and elsewhere.